cmd/proxy/config/apiConfig/credentials.toml

```diff
@@ -0,0 +1,9 @@
+# Credentials holds the list of username-pair pairs that allow access to certain endpoints.
+# Please change these example values as they are just placeholders.
+Credentials = [
+    { Username = "example", Password = "example" },
```

Comment:
Why not held already hashed? More secure.

Reply:
because it would be the same as passing the password through API in `plain text`. this way, a user has to hash his 'plain-text' password in order to gain access

Reply:
done